### DISCUSSION TITLE
How to invoke shutter position over http api call

### DISCUSSION THREAD

**Discussion by ciberp** [arendst/Tasmota#20107] [Q&A]
Hi!

Is there a way to invoke shutter position over http api call? In console I sometimes view:

`10:25:56.906 RSL: SENSOR = {"Time":"2023-11-26T10:25:56","Switch1":"ON","Switch2":"ON","Shutter1": "Position":100,"Direction":0,"Target":100,"Tilt":0}}`

I would like to get this info over http api if possible.

Thanks for the answer,
Regards

Peter

**Comment by sfromis** [comment]
The command `Status 10` provides same data as in sensor payloads; have not verified if it works for shutters too. And all commands can be sent via http.

Anyway, MQTT is recommended instead of http commands (the word "API" is overstating what it really is).
